chiranjeevi

```diff
@@ -1,2 +1,3 @@
 first cinema 
 second cinema
+third cinema
```

Comment:
tq for adding third line 
add fourth line also

Reply:
ok done

Reply:
cool can we resolve the commit radha

Reply:
ok lakshmi